polymorphic/admin/parentadmin.py

```diff
@@ -334,9 +315,9 @@ def subclass_view(self, request, path):
             try:
                 pos = path.find('/')
                 if pos == -1:
-                    object_id = long(path)
+                    object_id = int(path)
```

Comment:
I'm not so sure, as Django's `BigIntegerField` can produce longs on Python 2. I've reverted this in 67dd5c0a0dd33537c1deb946dbae0c8e98fbecb2